man/systemd.network.xml

```diff
@@ -1225,8 +1225,8 @@
           <term><varname>Protocol=</varname></term>
           <listitem>
             <para>The protocol identifier for the route. Takes a number between 0 and 255 or the special values
-            <literal>kernel</literal>, <literal>boot</literal> and <literal>static</literal>. Defaults to
-            <literal>static</literal>.
+            <literal>kernel</literal>, <literal>boot</literal>, <literal>static</literal>,
+            <literal>ra</literal> and <literal>dhcp</literal>. Defaults to <literal>static</literal>.
```

Comment:
We should not add it here in man. The debugging messages should automatically pick it . So adding in the table just fine. Not in the man.

Reply:
Initially I thought so. But considering about `kernel` and `boot`, I have no idea about the situation we need to set these values. Moreover, we can still set numeric values which are equivalent to `ra` or `dhcp`. So, I think there are no reason to list `ra` and `dhcp` in the man page.

Reply:
Which value we can set ? This is not how this works.

Reply:
Arbitrary value by numeric.

Reply:
This is actually letting the admins doing wrong configs. These two no meant for configurable . I do not agree with this patch.